src/test/java/org/java_websocket/framing/CloseFrameTest.java

```diff
@@ -42,7 +42,7 @@ public void testConstructor() {
         CloseFrame frame = new CloseFrame();
         assertEquals("Opcode must be equal", Opcode.CLOSING, frame.getOpcode());
         assertEquals("Fin must be set", true, frame.isFin());
-        assertEquals("TransferedMask must not be set", false, frame.getTransfereMasked());
+        assertEquals("transferredMask must not be set", false, frame.getTransfereMasked());
```

Comment:
Is it ok to fix? The field name in frame `FramedataImpl1` was `transferemasked`.